dependencyManagement/build.gradle.kts

```diff
@@ -31,7 +31,7 @@ val otelSnapshotVersion = "1.13.0"
 
 val DEPENDENCY_BOMS = listOf(
   "com.amazonaws:aws-java-sdk-bom:1.12.185",
-  "com.fasterxml.jackson:jackson-bom:2.13.2",
+  "com.fasterxml.jackson:jackson-bom:2.13.2.1",
```

Comment:
Thank you so much for the PR and detailed [description](https://mvnrepository.com/artifact/com.fasterxml.jackson/jackson-bom/2.13.2)
, it looks like the updated version is `2.13.2.2`. can we possibly use that version.

Reply:
Sure! updated the jackson bom to [2.13.2.20220328](https://mvnrepository.com/artifact/com.fasterxml.jackson/jackson-bom/2.13.2.20220328)

Reply:
![image](https://user-images.githubusercontent.com/55892686/162275460-1532552f-946e-4668-bf63-0bdab5dc7d8a.png)

Reply:
⭐ :star:

Reply:
@vasireddy99 Quick question on this, so I can loop back to my team. When is this version going to be available 😅

Reply:
Hi @jinwoov - the version specified in this repo is actually only used in tests, the actual update will happen after the next upstream javaagent release and then release of the ADOT version. This should happen in 1-2 weeks. Note that if you aren't using the javaagent with AWS Lambda, you aren't using the agent's jackson-databind and would be unaffected

Reply:
Hey @anuraaga, thanks for the update. I'll keep an eye out for the release. The issue with lower version of jackson-databind is more to do with Prisma picking up the vulnerability in the cluster. Hopefully new release will solve this problem. 😄